pkgs/applications/misc/fitnesstrax/default.nix

```diff
@@ -0,0 +1,44 @@
+{ pkgs,
+  fetchFromGitHub ? pkgs.fetchFromGitHub,
+  stdenv ? pkgs.stdenv,
+  rustPlatform ? pkgs.rustPlatform,
+  wrapGAppsHook ? pkgs.wrapGAppsHook,
+}:
+rustPlatform.buildRustPackage rec {
+  name = "fitnesstrax-${version}";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "luminescent-dreams";
+    repo = "fitnesstrax";
+    rev = "516a9e48978ac0f90bbbeade34d626211ccb78a3";
```

Comment:
do you have a tag in your repo? then you could do `rev = version;`

Reply:
Sweet. I did not know that was possible.

Reply:
Tagged at fitnesstrax-0.1.0!